src/main/java/net/pms/dlna/DVDISOTitle.java

```diff
@@ -109,80 +158,65 @@ public void run() {
 		String aspect = null;
 		String width = null;
 		String height = null;
-		ArrayList<DLNAMediaAudio> audio = new ArrayList<>();
-		ArrayList<DLNAMediaSubtitle> subs = new ArrayList<>();
+		String codecV = null;
+		ArrayList<DLNAMediaAudio> audioTracks = new ArrayList<>();
+		ArrayList<DLNAMediaSubtitle> subtitles = new ArrayList<>();
 		if (lines != null) {
 			for (String line : lines) {
 				if (line.startsWith("DVD start=")) {
 					nbsectors = Integer.parseInt(line.substring(line.lastIndexOf('=') + 1).trim());
-				}
-				if (line.startsWith("audio stream:")) {
-					DLNAMediaAudio lang = new DLNAMediaAudio();
-					lang.setId(Integer.parseInt(line.substring(line.indexOf("aid: ") + 5, line.lastIndexOf('.')).trim()));
-					lang.setLang(line.substring(line.indexOf("language: ") + 10, line.lastIndexOf(" aid")).trim());
-					int end = line.lastIndexOf(" langu");
-					if (line.lastIndexOf('(') < end && line.lastIndexOf('(') > line.indexOf("format: ")) {
-						end = line.lastIndexOf('(');
-					}
-					lang.setCodecA(line.substring(line.indexOf("format: ") + 8, end).trim());
-					if (line.contains("(stereo)")) {
-						lang.getAudioProperties().setNumberOfChannels(2);
-					} else {
-						lang.getAudioProperties().setNumberOfChannels(6);
+				} else if (line.startsWith("audio stream:")) {
```

Comment:
@Sami32 Now I understand what you mean. Its not the same even though both options work in this case. Using ```else``` is more efficient though, because once one of the if expressions has been evalutated to true, it doesn't have to evalute the rest. When skipping else, all the expressions must be evaluted for every  line.

That said, I doubt it makes any practical difference here since most of the parsed lines won't evaluate to true for any of them, which means that all will be evaluated in most cases anyway. More importantly I think it improves readability because it expresses to the reader that only one of these will be used for any one line.